env/.env.clinic

```
@@ -1,2 +1,2 @@
-VITE_USE_EEG="true"
-VITE_USE_PHOTODIODE="true"
\ No newline at end of file
+USE_EVENT_CODES="true"
```

We do have to prefix these variables with `VITE_` so they are loaded correctly with the compiler. You can read more here: https://vitejs.dev/guide/env-and-mode#env-files
```suggestion
VITE_USE_EVENT_CODES="true"
```